README.md

```diff
@@ -16,21 +16,20 @@ There are four options within the CLI:
 - `--main-pr <issueNumber>`       : Completes the annual cleanup by parsing issue and creating PR
 - `--validate <filePath> [--fix]` : Validates a given cnames_active file for perfect formatting
 
-When using `--perfect`, `--main-issue`, or `--main-pr`, you will need to have a `config.json` repo
+When using `--perfect`, `--main-issue`, or `--main-pr`, you will need to have a `config.json` file
 created in the root of the repository following `config.example.json`.
 
 If you are using `--validate`, then no config is needed as this operates against a given local file
 path rather than using the GitHub API. This option can also be passed a `--fix` flag after the file
 path to automatically fix any violations.
 
-### Example Generated Main Issue
+### Examples
 
+#### Generated Main Issue
 https://github.com/js-org-cleanup/simulated-js.org/issues/46
 
-### Example Generated Contact Issue
-
+#### Generated Contact Issue
 https://github.com/js-org-cleanup/simulated-automatic-contact/issues/12
 
-### Example Generated Pull Request
-
+#### Generated Pull Request
```

Comment:
You are still missing linebreaks between headings and content here

Reply:
Sorry, my bad